flowzone.yml

```diff
@@ -1699,25 +1730,17 @@ jobs:
         shell: bash --noprofile --norc -eo pipefail -x {0}
 
     steps:
-      # https://github.com/dawidd6/action-download-artifact
-      # TODO: what if this is a tag event and PR artifacts do not exist?
-      - name: Download source artifact from PR
-        uses: dawidd6/action-download-artifact@v2
-        with:
-          github_token: ${{ secrets.FLOWZONE_TOKEN }}
-          commit: ${{ github.event.pull_request.head.sha || github.event.head_commit.id }}
-          path: ${{ runner.temp }}
-          workflow_conclusion: success
-          name: source-${{ github.event.pull_request.head.sha || github.event.head_commit.id }}
-
+      - *downloadSourceArtifact
```

Comment:
I think `downloadSourceArtifact` only works within the context of the current action whereas this needs to download an artifact from another action?

Reply:
While `*downloadSourceArtifact` works here, the history will contain the merge commit and the versioning, commit, which means the release notes are not that nice, see https://github.com/balena-os/fatrw/releases/tag/v0.2.14 for an example. I'll revert to using the PR source in a new PR